downstream/modules/platform/proc-controller-api-browsing-api.adoc

```diff
@@ -0,0 +1,17 @@
+[id="controller-api-browsing"]
+
+.Procedure
+
+. Go to the {ControllerName} REST API in a web browser at: 
```

Comment:
Do you know  what the difference is between a REST API and a RESTful API? The user guide mentions RESTful.